site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -495,11 +485,11 @@ Duration: 1
 
 Once you have finished with the application and want to clean up your environment you can execute the following script
 
-```sql
-DROP APPLICATION APPL_MAPPING;
+```sh
+snow sql -q " DROP APPLICATION APPL_MAPPING;
```

Comment:
Instead of using SQL to drop the application + package, we need to use `snow app teardown`.